Homework_8_MyAmazingRPG/MyAmazingRPG/src/Character.java

```diff
@@ -0,0 +1,66 @@
+public class Character {
+    String name;                    //Ім'я персонажа.
+    int health;                     //Кількість очок здоров'я персонажа.
+    int strength;                   //Атрибут сили, який впливає на силу атаки.
+    int defense;                    //Атрибут захисту, який впливає на зменшення шкоди, коли об'єкт захищається.
+    static int characterCount;      //Статичне поле для відстеження кількості створених персонажів.
+
+    Character(String name, int health, int strength, int defense) {
+        this.name = name;
+        this.health = health;
+        this.strength = strength;
+        this.defense = defense;
+        characterCount++;
+    }
+
+
+
+    public int Attack(Character otherPerson) {
+        int damage;
+        damage = this.strength - otherPerson.defense;
+
+            if (damage < 0) {
+                damage = 0;
+            }
+
```

Comment:
here one thing is missing:
You have `otherPerson` (enemy) object passed as a method parameter, so you can directly call

`otherPerson.takeDamage(damage)`

so you won't be forced to call it explicitly in the client code (Main class)